### DISCUSSION TITLE
Disable Login / Default Login

### DISCUSSION THREAD

**Discussion by M1scer** [immich-app/immich#11998] [Q&A]
I would like to operate behind Authelia. My wife and I both use the same Immich login so far. Is there a way to be automatically logged in with an account after you have verified yourself via Authelia?

Or what alternative solutions are there to achieve something similar?

**Comment by bo0tzz** [comment]
Yes, this is all configurable in the oauth settings https://my.immich.app/admin/system-settings?isOpen=image+oauth+password.

https://immich.app/docs/administration/oauth

**Reply by M1scer** [reply]
I have configured authelia so i can login. But how can i force a specific account to be logged in?

Each Authelia account will try to login with a different username

**Reply by bo0tzz** [reply]
You mean you want different Authelia accounts to log in to the same Immich account? You'd have to configure that by setting custom user claims to be sent by Authelia.

**Reply by M1scer** [reply]
Yes exactly. Thats what i am looking for. I have not found if or how to do this with authelia.
Have you done that before?
Otherwise i will ask for help at authelia.

**Reply by superfunk2000** [reply]
@M1scer, did you solve it? I'm interested in the topic, too.

**Reply by M1scer** [reply]
@superfunk2000 sorry for my late response.
I have not found a solution. I now use a separate login for everyone and use partner sharing.
